inference-engine/src/preprocessing/ie_preprocess_gapi_kernels_simd_impl.hpp

```diff
@@ -783,6 +783,134 @@ inline void copyRow_32F_impl(const float in[], float out[], int length) {
     }
 }
 
+// Resize (bi-linear, 32FC1)
+static inline void calcRowLinear_32FC1(float *dst[],
+                                       const float *src0[],
+                                       const float *src1[],
+                                       const float  alpha[],
+                                       const int    mapsx[],
+                                       const float  beta[],
+                                       const Size& inSz,
+                                       const Size& outSz,
+                                               int lpi) {
+    bool xRatioEq1 = inSz.width == outSz.width;
```

Comment:
Only now I've got that this means x ratio equals to 1. I mistakenly have suggested previously to rename all `xRatioEq1` to `xRatioEq`...
May be we need to open low-low-low priority JIRA to fix this back...?

 I need to say that `xRatioEq1` is very unreadable name, may be we can replace it with `widthNotChanged`?
And also `yRatioEq1` to `heightNotChanged`?

Reply:
If you don't mind, I'd stay this task for refactoring

Reply:
Has such JIRA already created?

Reply:
yes, task for common refactoring had been created a long time ago.